templates/.env

```diff
@@ -4,6 +4,7 @@ GUARDIAN_SECRET_KEY=[generate by running: "openssl rand -base64 64"]
 ROUTER_SECRETS=[Unix Epoch Timestamp: openssl rand -base64 64]
 OUI=3
 SELF_HOSTED=true
+INITIAL_DC_BALANCE=10000
```

Comment:
what do you think about naming it INITIAL_ORG_GIFTED_DC?

Reply:
done!